Lib/test/test_socket.py

```diff
@@ -52,6 +53,35 @@
 except ImportError:
     _socket = None
 
+def skipForRefleakHuntinIf(condition, issueref):
```

Comment:
Maybe it is make sense to store it in ``test.support.refleak_helper``? It's can be useful in future in other situations like this